bps/templates/bps_flat.yaml

```diff
@@ -0,0 +1,11 @@
+pipelineYaml: ${CP_PIPE_DIR}/pipelines/${INSTRUMENT}/cpFlat.yaml
+
+project: ${TICKET}
+campaign: ${TICKET}
+
+payload:
+  payloadName: ${INSTRUMENT}_${TICKET}
+  output: ${USER_CALIB_PREFIX}${INSTRUMENT}/calib/${TICKET}/${TAG}/flatGen-${FLAT_BAND}.${RERUN}
+  butlerConfig: ${REPO}
+  inCollection: ${USER_CALIB_PREFIX}${INSTRUMENT}/calib/${TICKET}/${TAG},${RAW_COLLECTION},${CALIB_COLLECTIONS}
+  dataQuery: ${!SELECTION_FLAT_BAND}
```

Comment:
I think this exclamation point is a typo.

Reply:
This is bash "indirection" which should allow one template to be used for all the flats with different selection variables.  That's the idea, at least.  I haven't gotten to testing this part yet!